src/main/java/org/spongepowered/common/mixin/optimization/MixinWorldServer_Explosion.java

```diff
@@ -48,21 +50,23 @@
 @Mixin(value = WorldServer.class, priority = 1111)
 public abstract class MixinWorldServer_Explosion extends MixinWorld {
```

Comment:
@gabizou @JBYoshi Any reason why this mixin is on its own and not in MixinWorldServer?

Reply:
@Zidane because it's altering the mechanics of multiple explosions occurring at the same time. This goes about removing the point of sending explosion packets (reducing lag on clients), this was meddled with @amaranth a while back but I haven't verified whether it should be enabled by default now or not.